Cargo.toml

```diff
@@ -18,6 +18,5 @@ members = [
   "components/builder-worker",
   "components/github-api-client",
   "components/net",
-  "components/op",
-  "components/segment-api-client"
```

Comment:
`segment-api-client` should remain in this repo.  I think @christophermaier just moved it over here the other day?

Reply:
Yes, please leave it!

Reply:
oops - rebase fail